### ISSUE TITLE
[NEW] Swag Opportunity  Bagisto🎉 

### ISSUE THREAD

**Issue by debxrshi** [monizb/hacktoberfest-swag-list#169]
**Name Of Organization**
Bagisto
**Description**
For every 4 PR that gets accepted, you can adopt Baggy The Elephant along with cool swags

**Swag Items**
Pen, Stickers, stuffed toy
**Link to proof**
Link to the place where you saw this
https://webkul.com/meetups/hacktober-fest-bagisto-2022/

**Comment by debxrshi** [comment]
I can't open a new PR until my last one is approved

**Comment by thefool76** [comment]
> I can't open a new PR until my last one is approved
Bagisto is already  added on the live website
